cypress/test/campaignTests/Banner.cy.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import Banner from '../../../components/campaigns/Banner';
+
+describe('Banner Component', () => {
+  it('should not render the banner when the date is not within the valid range', () => {
+    const today = new Date();
+    const [day,  month, year] = [today.getUTCDate(),  today.getUTCMonth(),  today.getUTCFullYear()];
+    if (year > 2022 || month !== 10 || day < 6) {
+      mount(<Banner />);
```

Comment:
Use `beforeEach` here to mount the `Banner` Component

Reply:
For this I can't use BeforeEach since , for some parts in the code , I am rendering Banner after the clock is set , to get the Banner displayed like here ```
const mockDate = new Date(2021, 10, 12).getTime();
    cy.clock(mockDate);
    mount(<Banner />);
```